.changeset/swift-files-retire.md

```diff
@@ -0,0 +1,5 @@
+---
+"@astrojs/starlight": patch
+---
+
+Improve word wrapping for optimal display on smaller mobile screens
```

Comment:
Small tweak to explain the search modal context for people reading the changelog.
```suggestion
Fix word wrapping in search modal on narrow screens
```